MediaBrowser.Providers/Manager/ProviderManager.cs

```diff
@@ -157,6 +157,7 @@ public async Task SaveImage(BaseItem item, string url, ImageType type, int? imag
         {
             var httpClient = _httpClientFactory.CreateClient(NamedClient.Default);
             using var response = await httpClient.GetAsync(url, cancellationToken).ConfigureAwait(false);
+            response.EnsureSuccessStatusCode();
```

Comment:
In 99% of cases, you don't want to use this. It throws an exception.

You should revert back to your previous commit otherwise the image url will not be removed and it will simply try again next time.

Reply:
Thanks. Yea I see `ConvertImageToLocal` even checks for `Forbidden`, which is the exact case I was hitting.